### ISSUE TITLE
there is no "main" landmark region in the app

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/report-a-cybercrime#1254]
## Summary

Each page should have one main landmark.
The main landmark should be a top-level landmark.

## Steps to reproduce

The top level content container is a div with an id of "main" It is not identified as the main region. Fix in App.js
